contrib/codegen/templates/input/input_snapshot.gotmpl

```diff
@@ -376,19 +366,28 @@ func (b *multiCluster{{ $snapshotName }}Builder) insert{{ $kindPlural }}FromClus
 {{- end }}
 
 
-{{- else }}
-
 // build a snapshot from resources in a single cluster
 type singleCluster{{ $snapshotName }}Builder struct {
     mgr manager.Manager
+    clusterName string
 }
 
-// Produces snapshots of resources across all clusters defined in the ClusterSet
+// Produces snapshots of resources read from the manager for the given cluster
```

Comment:
```suggestion
// Produces snapshots of resources read from the manager for the local cluster
```

Reply:
Technically you can create one of these from any manager doesn't have to be the local